pkgs/data/misc/shared-mime-info/default.nix

```
@@ -10,14 +13,16 @@
 
 stdenv.mkDerivation rec {
   pname = "shared-mime-info";
-  version = "1.13.1";
+  version = "2.1";
 
   src = fetchurl {
-    url = "https://gitlab.freedesktop.org/xdg/${pname}/uploads/5349e18c86eb96eee258a5c1f19122d0/${pname}-${version}.tar.xz";
-    sha256 = "1bic8z5nz08qxv1x6zlxnx2j4cmlzm12kygrn3rrh1djqxdhma3f";
+    url = "https://gitlab.freedesktop.org/xdg/${pname}/uploads/0ee50652091363ab0d17e335e5e74fbe/${pname}-${version}.tar.xz";
```

```suggestion
    url = "https://gitlab.freedesktop.org/xdg/shared-mime-info/uploads/0ee50652091363ab0d17e335e5e74fbe/shared-mime-info-${version}.tar.xz";
```
Also why is this not using fetchFromGitLab?

---

Are you supposed to avoid pname? I'm very new to this so I have no idea. Also, I've adopted `fetchFromGitlab` now, using `${pname}`, but can change it if requested.